helm/cluster/README.md

```diff
@@ -490,6 +492,7 @@ Provider-specific properties that can be set by cluster-$provider chart in order
 | `providerIntegration.pauseProperties` | **Pause properties** - A map of property names and their values that will affect setting pause annotation|**Type:** `object`<br/>|
 | `providerIntegration.pauseProperties.*` |**None**|**Types:** `string, number, integer, boolean`<br/>|
 | `providerIntegration.provider` | **Provider** - The name of the Cluster API provider. The name here must match the name of the provider in cluster-<provider> app name.|**Type:** `string`<br/>|
+| `providerIntegration.registry` | **Registry** - The registry to use for pulling images. If not set, the default registry is used.||
```

Comment:
That looks weird, isn't the Type missing?

Reply:
could be becasue it can be null or object?

Reply:
fixed